README.md

```diff
@@ -1286,6 +1286,7 @@ Software that does not fit in another section.
 - [CapRover](https://caprover.com/) - Build your own PaaS in a few minutes. ([Demo](https://captain.server.demo.caprover.com/#/login), [Source Code](https://github.com/caprover/caprover)) `Apache-2.0` `Docker/Nodejs`
 - [changedetection.io](https://github.com/dgtlmoon/changedetection.io) - Self-hosted tool for staying up-to-date with web-site content changes. `Apache-2.0` `Python/Docker`
 - [CloudBeaver](https://cloudbeaver.io/) - Self-hosted management of databases, supports PostgreSQL, MySQL, SQLite and more. A web/hosted version of DBeaver. ([Source Code](https://github.com/dbeaver/cloudbeaver)) `Apache-2.0` `Nodejs`
+- [Cloudlog](https://cloudlog.co.uk) - Cloudlog is a self-hosted PHP application that allows you to log your amateur radio contacts anywhere. ([Demo](http://demo.cloudlog.co.uk/), [Source Code](https://github.com/magicbug/cloudlog)) `MIT` `PHP`
```

Comment:
- The main link should be https://www.magicbug.co.uk/cloudlog/
- The demo link does not work, points to a `Website Disabled` Virtualmin page, please remove the demo link.

Reply:
Thanks for the feedback! All resolved.